### PR TITLE
Update fixed.lua

### PR DESCRIPTION
**Pull request by legostax**
Adds more info to the docs for the underdocumented `wibox.layout.fixed.horizontal` and `wibox.layout.fixed.vertical`.

(hopefully I didn't screw anything up. I'm a noob to pull requests on Github)

### CODE REVIEW

File: lib/wibox/layout/fixed.lua [hunk 1]
```diff
@@ -289,6 +289,7 @@ end
 --- Returns a new horizontal fixed layout. Each widget will get as much space as it
 -- asks for and each widget will be drawn next to its neighboring widget.
 -- Widgets can be added via :add() or as arguments to this function.
+-- Note that widgets ignore forced_height.
```
Comment (Elv13) [inline lib/wibox/layout/fixed.lua]: please add the "`" around `forced_height`

Also, maybe rephrase this a little to make it clear that the widget use the preferred/minimum value in one axis and expand the other axis so the widget takes all of [that axis] space.

**Comment by legostax** [comment]
Updated for review.

**Comment by blueyed** [comment]
LGTM, should be squash-merged (we'll handle that).
